src/index.ts

```diff
@@ -0,0 +1,211 @@
+import {
+  Beautifier,
+  Language,
+  BeautifierBeautifyData,
+  DependencyType,
+  ExecutableDependency,
+} from "unibeautify";
+import * as readPkgUp from "read-pkg-up";
+import * as tmp from "tmp";
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+
+const { pkg } = readPkgUp.sync({ cwd: __dirname });
+
+export const beautifier: Beautifier = {
+  name: "clang-format",
+  package: pkg,
+  badges: [
+    {
+      description: "Build Status",
+      url:
+        "https://travis-ci.com/Unibeautify/beautifier-clang-format.svg?branch=master",
+      href: "https://travis-ci.com/Unibeautify/beautifier-clang-format",
+    },
+    {
+      description: "Test Coverage",
+      url:
+        "https://api.codeclimate.com/v1/badges/8b4e5b19de96f4744ffa/test_coverage",
+      href:
+        "https://codeclimate.com/github/Unibeautify/beautifier-clang-format/test_covera" +
+        "ge",
+    },
+    {
+      description: "Maintainability",
+      url:
+        "https://api.codeclimate.com/v1/badges/8b4e5b19de96f4744ffa/maintainability",
+      href:
+        "https://codeclimate.com/github/Unibeautify/beautifier-clang-format/maintainabi" +
+        "lity",
+    },
+    {
+      description: "Greenkeeper",
+      url:
+        "https://badges.greenkeeper.io/Unibeautify/beautifier-clang-format.svg",
+      href: "https://greenkeeper.io/",
+    },
+  ],
+  options: {
+    "C": true,
+    "C++": true,
+    Java: true,
+    "Objective-C": true
+  },
+  dependencies: [
+    {
+      type: DependencyType.Executable,
+      name: "clang-format",
+      program: "clang-format",
+      parseVersion: [/clang-format version (\d+\.\d+\.\d+)/],
+      homepageUrl: "https://clang.llvm.org/docs/ClangFormat.html",
+      installationUrl: "https://clang.llvm.org/docs/ClangFormat.html",
+      bugsUrl: "https://clang.llvm.org/docs/ClangFormat.html",
+      badges: [
+        {
+          description: "",
+          url: "",
+          href: "",
```

Comment:
The dependency's badges should either be empty of a valid badge.

I could not find any badges for ClangFormat, so maybe empty array is applicable here.